lib/ansible/module_utils/urls.py

```diff
@@ -541,8 +556,8 @@ def build_ssl_validation_error(hostname, port, paths, exc=None):
     if not HAS_SSLCONTEXT:
         msg.append('If the website serving the url uses SNI you need'
                    ' python >= 2.7.9 on your managed machine')
-        if not HAS_URLLIB3_SNI_SUPPORT:
-            msg.append('or you can install the `urllib3`, `pyopenssl`,'
+        if not HAS_URLLIB3_PYOPENSSLCONTEXT or not HAS_URLLIB3_SSL_WRAP_SOCKET:
+            msg.append('or you can install the `urllib3`, `pyOpenSSL`,'
```

Comment:
I believe this message should be updated to reflect the latest workaround guidance from `urllib` found [here](https://urllib3.readthedocs.io/en/latest/user-guide.html#ssl-py2), to something like 'or you can install the `urllib3` package with the `secure` extra (`urllib3[secure]`), or the `cryptography`, `pyopenssl`, `idna`, and `certifi` packages that the `secure` extra bundles.'